tensorboard/plugins/npmi/http_api.md

```diff
@@ -121,9 +102,8 @@ Example:
       -0.17293912172317505,
       -0.11519329994916916,
       -0.05774582177400589,
-      0.11519329994916916
-  ],
-  "annotation_2": [
+      0.11519329994916916]
```

Comment:
missing a comma for this to be valid json

Reply:
Fixed. Can you merge this? I do not have the rights to do so anymore.

Reply:
Yep, done